src/averageAges.js

```diff
@@ -36,8 +61,13 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const woman = withChildren
+    ? people.filter((person, index, arr) => isMother(person, index, arr))
```

Comment:
`isMother` function accepts same arguments as filter callback so you could just pass the `isMother` similarly to the `isWoman`
```
people.filter(isMother)
```
instead of writing callback arrow function that passes same arguments to the `isMother`